bears/vcs/git/CISkipInspectBear.py

```diff
@@ -0,0 +1,60 @@
+import re
+
+from bears.vcs.git.GitCommitMetadataBear import GitCommitMetadataBear
+from coalib.bears.GlobalBear import GlobalBear
+from coalib.results.Result import Result
+
+
+class CISkipInspectBear(GlobalBear):
+    LANGUAGES = {'Git'}
+    AUTHORS = {'The coala developers'}
+    AUTHORS_EMAILS = {'coala-devel@googlegroups.com'}
+    LICENSE = 'AGPL-3.0'
+
+    BEAR_DEPS = {GitCommitMetadataBear}
+
+    def run(self, dependency_results, ci_engines_used,
+            filename_patterns, **kwargs):
+        """
+        Check the HEAD commit to see if it disables CI build and
+        return result accordingly.
+
+        :param ci_engines_used:     CI engines used in the project.
+        :param filename_patterns:   To identify files that should not
+                                    skip CI builds.
+        """
+        for result in dependency_results[GitCommitMetadataBear.name]:
+
+            for ci_engine in ci_engines_used:
+
+                supported_regex = r'\[ci skip\]|\[skip ci\]'
+
+                # AppVeyor allows to skip build by commit if the
+                # supported sequence is present in the title of the
+                # commit only. All other CI tools allow these sequences
+                # to be present anywhere in the commit message.
+                if ci_engine == 'AppveyorCI':
+                    pos = result.raw_commit_message.find('\n')
+                    commit_title = result.raw_commit_message[:pos] if (
+                        pos != -1) else result.raw_commit_message
+
+                    match = re.search(supported_regex, commit_title)
+
+                else:
+                    match = re.search(
+                        supported_regex, result.raw_commit_message)
+
+                if match:
+
+                    all_files = (result.modified_files +
+                                 result.added_files + result.deleted_files)
+
+                    for file in all_files:
+                        for pattern in filename_patterns:
+                            if re.search(pattern, file):
+                                yield Result(
+                                    self,
+                                    'This commit modifies files of type '
+                                    '\"%s\" and cannot disable CI build.' %
```

Comment:
It's not necessary to escape `"`

`..., thus should not disable CI build.`
Because I think this bear only can advice the user (we'd place it in GitMate or somewhere), if the CI didn't run because of the "skip" commit then this bear didn't run as well